core/src/main/java/org/elasticsearch/common/xcontent/cbor/CborXContent.java

```diff
@@ -77,6 +77,11 @@ public XContentGenerator createGenerator(OutputStream os, String[] filters) thro
     }
 
     @Override
+    public XContentGenerator createGenerator(OutputStream os, String[] filters, boolean inclusiveFilters) throws IOException {
```

Comment:
Same here, `createGenerator(OutputStream os, String[] filters)` could now use this method with inclusive set to true?